docs/basics/integrations/webhooks.md

```diff
@@ -0,0 +1,193 @@
+---
+id: webhooks
+title: Webhooks Integration
+sidebar_label: Webhooks
+description: Setup a webhook to which Sauce Labs test results are automatically pushed.
+keywords:
+- get-test-results
+- webhooks
+- how-to
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><span className="sauceDBlue">COMING SOON</span></p>
+
+The Sauce Labs Webhooks integration allows you specify a URL to which Sauce Labs will push test result data as it becomes available, rather than forcing you to constantly poll for updates.
+
+## What You'll Need
+
+* A Sauce Labs account (if you don't have one, start a [free trial](https://saucelabs.com/sign-up))
+* Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
+* A URL able to consume events data pushed from Sauce Labs
+
+
+## Install the Webhooks Plugin
+
+1. From your Sauce Labs account, navigate to the [Account Integrations](https://app.staging.saucelabs.net/integrations) page.
+1. Click the Webhooks **Install** button.
+1. Enter the URL of the endpoint to which you would like Sauce Labs events pushed.
+1. Select the events you wish to receive:
+    * **VDC Events**: Receive test result events for jobs run on Sauce Labs desktop browsers and mobile emulators and simulators.
+    * **RDC Events**: Receive test result events for jobs run on Sauce Labs real devices.
+    * **Concurrency Usage**: Receive update events related to the number of sessions currently in use by your Sauce Labs organization and team.
+1. Click **Save** to complete your integration.
+
+
+<!-- Question1: Can any user go through the integration process multiple times to set up multiple webhooks? Will the Integrations page show the list of currently active webhooks? -->
+
+<!-- Question2: Is this integration data center specific? -->
+
+## Handle Event Data
+
+Each of the three events to which you can subscribe push a data object to the webhook endpoint you have specified. You should configure your endpoint to:
+
+* Accept a POST webhook request method from Sauce Labs
+* Check the event object to validate its type and parse the payload contained in the request object
+* Return a `2xx` success response to the request to close the call.
```

Comment:
Not sure if this part is needed, as it might be very implementation specific on user side. These steps may apply to a small number of cases, but in general no additional action should be required.